src/FileCache.php

```diff
@@ -135,8 +126,10 @@ public function set(string $key, mixed $value, null|int|DateInterval $ttl = null
         $file = $this->getCacheFile($key);
         $cacheDirectory = dirname($file);
 
-        if (!is_dir($this->cachePath) || ($this->directoryLevel > 0 && !$this->createDirectoryIfNotExists($cacheDirectory))) {
-            throw new CacheException("Failed to create cache directory \"{$cacheDirectory}\".");
+        if (!is_dir($this->cachePath)
+            || ($this->directoryLevel > 0 && !$this->createDirectoryIfNotExists($cacheDirectory))
```

Comment:
```suggestion
            || $this->directoryLevel > 0 && !$this->createDirectoryIfNotExists($cacheDirectory)
```

Reply:
With brackets code more clarify.

Reply:
You clarified this when you moved it to a new line.